bittensor/_neuron/text/advanced_server/nucleus_impl.py

```diff
@@ -221,7 +221,7 @@ def config ():
         parser.add_argument('--neuron.metagraph_sync', type=float, help='how often to sync the metagraph', default=100000)
         parser.add_argument('--neuron.blocks_per_set_weights', type=float, help='how often to sync set weights', default=100)
         parser.add_argument('--neuron.blocks_per_epoch', type=int, help='Blocks per epoch', default=2)
-        parser.add_argument('--neuron.blacklist.time', type=int, help='how often a peer can query you (seconds) ', default=2)
+        parser.add_argument('--neuron.blacklist.time', type=int, help='how often a peer can query you (seconds) ', default=0)
```

Comment:
Maybe later on we should add a 'frequency of request' to our priority function 
as an anti-measure for this change.

Reply:
Yea, I agree a time component to the priority function makes more sense